leapsectz/leapsectz_test.go

```diff
@@ -58,3 +58,82 @@ func TestParse(t *testing.T) {
 		t.Errorf("wrong leap second count")
 	}
 }
+
+func TestParseV2(t *testing.T) {
+	byteData := []byte{
+		'T', 'Z', 'i', 'f', // magic
+		'2', 0x00, 0x00, 0x00, // version
+		0x00, 0x00, 0x00, 0x00, // pad
+		0x00, 0x00, 0x00, 0x00, // pad
+		0x00, 0x00, 0x00, 0x00, // pad
+		0x00, 0x00, 0x00, 0x00, // UTC/local
+		0x00, 0x00, 0x00, 0x00, // standard/wall
+		0x00, 0x00, 0x00, 0x01, // leap
+		0x00, 0x00, 0x00, 0x00, // transition
+		0x00, 0x00, 0x00, 0x00, // local tz
+		0x00, 0x00, 0x00, 0x00, // characters
+		0x04, 0xb2, 0x58, 0x00, // leap time
+		0x00, 0x00, 0x00, 0x01, // leap count
+		'T', 'Z', 'i', 'f', // magic
+		'2', 0x00, 0x00, 0x00, // version
+		0x00, 0x00, 0x00, 0x00, // pad
+		0x00, 0x00, 0x00, 0x00, // pad
+		0x00, 0x00, 0x00, 0x00, // pad
+		0x00, 0x00, 0x00, 0x00, // UTC/local
+		0x00, 0x00, 0x00, 0x00, // standard/wall
+		0x00, 0x00, 0x00, 0x01, // leap
+		0x00, 0x00, 0x00, 0x00, // transition
+		0x00, 0x00, 0x00, 0x00, // local tz
+		0x00, 0x00, 0x00, 0x00, // characters
+		0x00, 0x00, 0x00, 0x00, // leap time (first 32 bits)
+		0x04, 0xb2, 0x58, 0x00, // leap time (last 32 bits)
+		0x00, 0x00, 0x00, 0x01, // leap count
```

Comment:
I would very much like to see a test for 2+ leap seconds, just to make sure we handle it correctly.